src/components/app/App.tsx

```diff
@@ -35,6 +36,7 @@ const App: React.FC = () => {
     <Provider store={store}>
       <ThemeProvider theme={theme}>
         <Switch>
+          <Route path="/play/pallet" component={PreLoginPage}></Route>
```

Comment:
Can you just make this /login
?

Reply:
@lindsaymacvean There is already page for login. This is pre login page maybe /pre-login?